numba/pycc/platform.py

```diff
@@ -57,7 +58,7 @@ def _check_external_compiler():
                 ntf.close()
                 # *output_dir* is set to avoid the compiler putting temp files
                 # in the current directory.
-                compiler.compile([ntf.name], output_dir=gettempdir())
```

Comment:
Does removing this mean there's no need to import `gettempdir` now?

Reply:
doesn't look like it, I've removed unused imports in my latest commit